### ISSUE TITLE
Deno std does not import process

### ISSUE THREAD

**Issue by bershanskiy** [denoland/deno#17751]
Node has a global `process` module/variable which does not need importing but can be imported from `process`:
```js
import process from 'node:process';
```
In deno (as of `1.30.0`), one does need to import it. Turns out, some of deno's std modules use `process` but do not explicitly import it.

## Repro steps:
This repro attempts to trigger a warning by opening the same file a bunch of times, but instead causes an exception to be thrown `Uncaught ReferenceError: process is not defined`.
### `demo.js`
```js
import fs from 'fs';

for (let i = 0; i < 100; i++) {
  fs.watchFile('./deno.json', {}, () => {});
}
```

### `deno.json` 
```json
{ "imports": { "fs": "node:fs" }}
```

### Run command
```
deno run --allow-read=deno.json demo.js
```

**Comment by bershanskiy** [comment]
This is the exact line in deno_std:
https://github.com/denoland/deno_std/blob/main/node/_events.mjs#L464

Also, I realize that the code lives in `deno_std`, but I'm not sure whether the bug entirely resides there since just importing `process` into, say `fs` would probably be an overkill.

**Comment by bartlomieju** [comment]
Thanks, good catch. We're in process of moving this code into `ext/node` so it should be fixed there once https://github.com/denoland/deno/pull/17724 lands.

**Comment by bartlomieju** [comment]
This problem will be fixed in Deno v1.31. Any import to built-in Node module ensures that `process` is available.
